### PR TITLE
Fixed Warnings

### PR DESCRIPTION
**Pull request by vigbalu**
Fixed few warnings in ompd and gdb-plugin code.

### CODE REVIEW

**Review by jinisusan — APPROVED** [review]
LGTM

File: openmp/libompd/gdb-plugin/ompdModule.c [hunk 1]
```diff
@@ -574,7 +574,7 @@ ompd_rc_t _read_string (
 	}
 	Py_ssize_t retSize;
 	const char* strbuffer = PyUnicode_AsUTF8AndSize(retString, &retSize);
-	if(retSize >= nbytes) {
+	if((unsigned int)retSize >= nbytes) {
```
Comment (jprotze) [inline openmp/libompd/gdb-plugin/ompdModule.c:577]: > The returned buffer always has an extra null byte appended (not included in size), regardless of whether there are any other null code points.
I think this should be:
```suggestion
	if((ompd_size_t)retSize+1 >= nbytes) {
```

File: openmp/libompd/gdb-plugin/ompdModule.c [hunk 2]
```diff
@@ -542,7 +542,7 @@ ompd_rc_t _read (
 		}
 		Py_ssize_t retSize = PyByteArray_Size(retArray);
 		const char* strBuf = PyByteArray_AsString(retArray);
-		if(retSize != nbytes) {
+		if((unsigned int)retSize != nbytes) {
```
Comment (jprotze) [inline openmp/libompd/gdb-plugin/ompdModule.c:545]:
```suggestion
		if((ompd_size_t)retSize != nbytes) {
```

**Comment by vigbalu** [comment]
Created a new pull request #38 with review comments addressed.

**Pull request closed without merging**